MCprep_addon/materials/default_materials.py

```diff
@@ -1,219 +0,0 @@
-	sync_file = sync.get_sync_blend(context)
-
-	init_mats = list(bpy.data.materials)
-	path = os.path.join(sync_file, "Material")
-	util.bAppendLink(path, import_name, False)  # No linking.
-
-	imported = set(list(bpy.data.materials)) - set(init_mats)
-	if not imported:
-		return f"Could not import {material.name}"
-	new_default_material = list(imported)[0]
-
-	# Checking if there's a node with the label Texture.
-	new_material_nodes = new_default_material.node_tree.nodes
-	if not new_material_nodes.get("MCPREP_diffuse"):
-		return "Material has no MCPREP_diffuse node"
-
-	if not material.node_tree.nodes:
-		return "Material has no nodes"
-
-	# Change the texture.
-	new_default_material_nodes = new_default_material.node_tree.nodes
-	material_nodes = material.node_tree.nodes
-
-	if not material_nodes.get("Image Texture"):
-		return "Material has no Image Texture node"
-
-	default_texture_node = new_default_material_nodes.get("MCPREP_diffuse")
-	image_texture = material_nodes.get("Image Texture").image.name
-	texture_file = bpy.data.images.get(image_texture)
-	default_texture_node.image = texture_file
-
-	if engine == "CYCLES" or engine == "BLENDER_EEVEE":
-		default_texture_node.interpolation = 'Closest'
-
-	material.user_remap(new_default_material)
-
-	# remove the old material since we're changing the default and we don't
-	# want to overwhelm users
-	bpy.data.materials.remove(material)
-	new_default_material.name = material.name
-	return None
-
-
-class MCPREP_OT_default_material(bpy.types.Operator):
-	bl_idname = "mcprep.sync_default_materials"
-	bl_label = "Sync Default Materials"
-	bl_options = {'REGISTER', 'UNDO'}
-
-	use_pbr: bpy.props.BoolProperty(
-		name="Use PBR",
-		description="Use PBR or not",
-		default=False)
-
-	engine: bpy.props.StringProperty(
-		name="engine To Use",
-		description="Defines the engine to use",
-		default="CYCLES")
-
-	SIMPLE = "simple"
-	PBR = "pbr"
-
-	track_function = "sync_default_materials"
-	track_param = None
-	@tracking.report_error
-	def execute(self, context):
-		# Sync file stuff.
-		sync_file = sync.get_sync_blend(context)
-		if not os.path.isfile(sync_file):
-			self.report({'ERROR'}, f"Sync file not found: {sync_file}")
-			return {'CANCELLED'}
-
-		if sync_file == bpy.data.filepath:
-			return {'CANCELLED'}
-
-		# Find the default material.
-		workflow = self.SIMPLE if not self.use_pbr else self.PBR
-		material_name = material_name = f"default_{workflow}_{self.engine.lower()}"
-		if not default_material_in_sync_library(material_name, context):
-			self.report({'ERROR'}, "No default material found")
-			return {'CANCELLED'}
-
-		# Sync materials.
-		mat_list = list(bpy.data.materials)
-		for mat in mat_list:
-			try:
-				err = sync_default_material(context, mat, material_name, self.engine.upper()) # no linking
-				if err:
-					env.log(err)
-			except Exception as e:
-				print(e)
-
-		return {'FINISHED'}
-
-
-class MCPREP_OT_create_default_material(bpy.types.Operator):
```

Comment:
Just sanity checking - I see the code is commented out in the UI, was this ever exposed such that users could have been using these material operators from this file? (outside of manually stumbling onto the code).

I just want o ensure it's safe to delete - I'm all for cleaning up dead code, just want to make sure I understand if there's any loss of functionality we need to communicate